test/core/circuit-relay.spec.js

```diff
@@ -102,9 +101,7 @@ describe.skip('circuit', function () {
 
   after((done) => factory.dismantle(done))
 
-  // TODO: 1) figure out why this test hangs randomly
-  // TODO: 2) move this test to the interop batch
-  it.skip('node1 <-> goRelay <-> node2', (done) => {
+  it('node1 <-> goRelay <-> node2', (done) => {
```

Comment:
Please move this test to the interop suite. That was what was in the TODO

Reply:
I can move those to the interop tests, but we'll lose any sort of browser coverage until ipfsd-ctl is ready.